config/relayer/config.go

```diff
@@ -76,6 +77,9 @@ func (c *RawRelayerConfig) Validate() error {
 	if c.MpcConfig.TopologyConfiguration.SecKey == "" {
 		return errors.New("topology configuration secret key not provided")
 	}
+	if c.MpcConfig.TopologyConfiguration.EncryptionKey == "" {
```

Comment:
Do we also need to validate the key length, as from looking into [documentation](https://pkg.go.dev/crypto/aes#NewCipher) I would say that the key must match one of the predefined sizes?